### PR TITLE
Drafted section on Overall Grounding Concept [9.1-9.2]

### PR DESCRIPTION
**Pull request by aehoppe**

### CODE REVIEW

File: esf_main.tex [hunk 1]
```diff
@@ -106,13 +106,22 @@ \section*{List of Abbreviations}
...
+
+The tractive system consists of a custom-designed and fabricated accumulator container which houses 72 LG Chem proprietary pouch cells, as well as a Rinehart Motion Systems PM100DX motor controller driving an Emrax 228 Medium Voltage outrunner motor. Communication to the motor controllers is achieved via the CAN network built into the low voltage system.
+
+The low voltage system consists primarily of a shutdown circuit and a series of peripheral sensors to gather data about vehicle state and performance, all powered by a 12V nominal sealed lead acid battery that is recharged by a DC/DC converter. The Shutdown circuit monitors the vehicle for dangerous conditions such as impacts or isolation faults of the Tractive System, and de-energizes the vehicle in case of emergency. The low voltage system also contains a CAN communucation network between Atmel ATmega16m1 microconttrollers that fulfill several rules-required functions like the Ready-to-Drive sound, as well as collecting data and serving as a debugging tool for both high and low power electrical systems.
```
Comment (byronwasti) [inline esf_main.tex:120]: I don't know how I feel about the 12V lead-acid battery for the GLV system. The one we used last year was incredibly bulky. It is fine in the ESF for now

File: esf_main.tex [hunk 2]
```diff
@@ -806,11 +822,12 @@ \subsection{LV part 2}
 \section{Overall Grounding Concept}
 \subsection{Description of the Grounding Concept}
 %Describe how you intend to achieve the resistances between components at the required levels as defined in EV4.3.
-TODO
+The vehicle's steel chassis is uses as GLVS ground. This ground will be established inside the master switch panel enclosure by attachment to a piositively-retained ring terminal on a chassis member that has been locally stripped of paint. All components of the GLVS will be grounded to this point on the chassis through the wiring harness for simplicity of installation and mechanical robustness. All conductive mechanical enlosures are mounted to the chassis with conductive metal fasteners. Electrical connection of the suspension members and knuckles to the main vehicle frame will be achieved by short jumpers with ring terminals on either end in the suspension member bolt stackup on either side of the spherical bearing.
```
Comment (byronwasti) [inline esf_main.tex:854]: > "The vehicle's steel chassis is **used** as GLVS ground."
Quick typo fix.

We might also want to have a grounding point at the front of the chassis as well, just to ensure that no part of the chassis floats above ground.

**Review by byronwasti — APPROVED** [review]
Just one small typo. Otherwise looks good for the ESF.

**Comment by byronwasti** [comment]
Also, you'll want to fix the merge conflicts before merging. They look pretty straightforward to fix.

**Comment by aehoppe** [comment]
Got it.

**Pull request merged**